packages/blockchain-wallet-v4-frontend/src/modals/Nfts/NftOrder/MakeOffer/WrapEth.fees.tsx

```diff
@@ -53,10 +53,14 @@ const Fees: React.FC<Props> = (props) => {
               <div>
                 <RightAlign>
                   <CoinDisplay size='14px' color='black' weight={600} coin='ETH'>
-                    {new BigNumber(val.totalFees).multipliedBy(val.gasPrice).toString()}
+                    {canWrap && needsWrap
+                      ? new BigNumber(val.totalFees).multipliedBy(val.gasPrice).toString()
```

Comment:
`val` could also be using destructuring

Reply:
Also it seems that` new BigNumber(val.totalFees).multipliedBy(val.gasPrice).toString()` is duplicated several times, you could simplify creating a constant an using it each time

Reply:
lets leave for now, thanks!